site/en/docs/privacy-sandbox/faq/index.md

```diff
@@ -304,6 +304,12 @@ the following properties:
 *  `auctionSignals`, a property of the auction configuration object passed to `navigator.runAdAuction()`
 *  `userBiddingSignals`, a property of the interest group configuration
    object passed to `navigator.joinAdInterestGroup()`
+   
+### What is the limit to the number of interest groups that a group owner (origin) can add a single browser to?
```

Comment:
```suggestion
### What is an owner's interest group limit for a single browser? {: #interest-group-limit }
```

Reply:
Maybe something like this: 'What is the maximum number of interest groups per group owner for a single user?'

(The use of 'browser' is good, to emphasise that PS APIs work with the 'browser' not the 'user', but in this context it might seem to imply a 'browser' in the sense of Chrome versus Edge.)